docs/tutorials/dataflows/chatbot.rst

```diff
@@ -22,32 +22,50 @@ messages which are directed to our bot.
     `examples/dataflow/chatbot <https://github.com/intel/dffml/blob/master/examples/dataflow/chatbot>`_
     directory of the DFFML source code.
 
+You'll need to install `aiohttp <https://github.com/aio-libs/aiohttp>`_
+and ``dffml-model-scikit`` (The model used for prediction).
+
+.. code-block:: console
+
+    $ pip install aiohttp dffml-model-scikit
+
 We'll write the operations for this dataflow in operations.py
 
+Adding necessary imports and defining ``Definitions`` for operation
+inputs.
+
 .. literalinclude:: /../examples/dataflow/chatbot/operations.py
-    :lines: 24-51
+    :lines: 1-16
+
+Defining config for our operations
+
+.. literalinclude:: /../examples/dataflow/chatbot/operations.py
+    :lines: 19-21
 
-All requests to Gitter's API requires the room id for our room.
+All requests to Gitter's API requires the room id of our room.
 ``get_room_id`` gets the ``room id`` from room name (The input to
 our dataflow).
 
 .. literalinclude:: /../examples/dataflow/chatbot/operations.py
-    :lines: 52-87
+    :lines: 24-51
 
 We listen to new messages directed to our bot.
 
 .. literalinclude:: /../examples/dataflow/chatbot/operations.py
-    :lines: 90-122
+    :lines: 52-87
 
 We'll use this op to send replies back to the chatroom
 
 .. literalinclude:: /../examples/dataflow/chatbot/operations.py
-    :lines: 125-220
+    :lines: 90-122
 
 This is the operation where all the logic for interpreting the messages
 go. If you have a Natural Language Understanding module It'd go here, so
 that you can parse unstructered data.
```

Comment:
I don't think we need `sys.stdout.flush()`, we're still not talking about why / how were using `CLI`. It would be good to simplify this further, so maybe don't let the user choose the name or the kind of model (scikit whatever).